app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -145,9 +145,9 @@ def handle_invalid_otp(type:, context: nil)
   def invalid_otp_error(type)
     case type
     when 'otp'
-      t('two_factor_authentication.invalid_otp')
+      t('two_factor_authentication.invalid_otp_html', count: attempts_count_remaining)
```

Comment:
Question I have, Since this is a common pattern/lockout for bad attempst with many of the other MFA attempts, is it worth it to add this pattern to all methods listed. Also right now the specs ask for just on the last attempt remaining display the attempts left. I can modify that pretty easily to show all attempts left.

Reply:
> Since this is a common pattern/lockout for bad attempst with many of the other MFA attempts, is it worth it to add this pattern to all methods listed.

I do think it'd be nice to be consistent. I'd leave it up to you if you want to address it here or create a follow-up ticket to bring those others in alignment. Maybe it ends up being simpler to just be consistent?

Reply:
Yea I think its simpler if we have the language we want content wise, and if its easy append it to all methods listed here in invalid_otp_error